definitions/infra-awstextract/dashboard.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "AWS Textract",
+  "description": null,
+  "pages": [
+    {
+      "name": "AWS Textract",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Response Time",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 6,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT max(`aws.textract.ResponseTime`)as max, min(`aws.textract.ResponseTime`)as min, average(`aws.textract.ResponseTime`)as avg FROM Metric SINCE 1 week AGO TIMESERIES"
```

Comment:
I would remove the SINCE from the query.
Usually, time must not be provided as we want to query data for the user's active time interval.
If SINCE is provided, `ignoreTimeRange` must be set to true, but must be a good reason to do that, as it leaves less flexibility to the user.